openadapt/app/dashboard/app/recordings/page.tsx

```diff
@@ -50,7 +50,7 @@ export default function Recordings() {
     }, []);
     return (
         <Box>
-            {/* {recordingStatus === RecordingStatus.RECORDING && (
+            {recordingStatus === RecordingStatus.RECORDING && (
```

Comment:
@KIRA009 is this intentional? It seems unrelated to the rest of the PR.

Reply:
@abrichr Yes, since the reason for commenting this out has been fixed, I have added it back